engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -739,6 +871,35 @@ else if (propertyName.equals(entry.getKey())) {
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/properties")
```

Comment:
The point was that we already have the API to list properties for an operator, we don't need to duplicate it. Extensions should be added only for those aspects that are specific to modules.